active/NumberBox/NumberBox.md

```diff
@@ -159,24 +185,31 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String Text; 
     String PlaceholderText;
 
+<!-- todo: Why does this say "basic"? Is there also an advanced validation mode? -->   
     NumberBoxBasicValidationMode BasicValidationMode;
 
     Boolean AcceptsCalculation;
```

Comment:
Had a comment on my last PR about whether this should be "AcceptCalculations", plural. I also would like to through "AcceptsExpressions" in the ring because I think it's more accurate.

Reply:
Not a deal breaker IMO, but since this is an API, an unqualified "expression" might be confused with an [ExpressionAnimation](https://docs.microsoft.com/uwp/api/Windows.UI.Composition.ExpressionAnimation).

Reply:
Agreed that this should at least be a plural "Calculations" if that is the route taken.

@MikeHillberg, would "AcceptsNumericalExpressions" be more clear?

Reply:
A compound calculation [is a calculation](https://en.wikipedia.org/wiki/Calculation): *For example, multiplying 7 by 6 is a simple algorithmic calculation. Estimating the fair price for financial instruments using the Black–Scholes model is a more complex algorithmic calculation.*

And the current naming has the advantage of matching TextBox.AcceptsReturn.

Reply:
AcceptsNumericalExpressions is long, and good point about AcceptsReturn. I like AcceptsCalculation.

Reply:
I will leave this as is and mark "Resolved."